src/types/Todo.ts

```diff
@@ -0,0 +1,5 @@
+export interface Todo {
+  id: number,
```

Comment:
Use semicolons instead of commas to separate properties in TypeScript interfaces